schoolopy/main.py

```diff
@@ -1891,15 +1893,44 @@ def get_assignment_comment(self, section_id, assignment_id, comment_id):
     # TODO: Support Web Content Package
     # TODO: Support Completion
 
+    def get_section_grades(self, section_id, query_name=None, query_value=None):
+        """
+        Returns a list of grades (paged).
+
+        :param section_id: ID of course's section.
+        :return: a list of grades for that section.
+
+        The following optional query strings can be appended to the path
+        to filter results:
+
+        :param query_name (timestamp, assignment_id, or enrollment_id)
+        :param query_value
+
+        timestamp:      return only grades that have been changed since
+                        the given timestamp, according to the server time.
+                        e.g. "timestamp": 1517551200
+        assignment_id:  filter grades for a given assignment
+        enrollment_id:  filter grades for a given enrollment
+        """
+
+        query = ''
+        if query_name and query_value:
+            query = '&{}={}'.format(query_name, query_value)
```

Comment:
Wouldn't this only allow for one of the possible queries to be used per request?

Reply:
Oh yeah, this too.